src/api-legacy/v1/borehole/export/admin.py

```diff
@@ -132,29 +132,29 @@ async def post(self, *args, **kwargs):
 
                         not_found = []
 
-                        for row in rows:
+                        # for row in rows:
 
-                            if row is None:
-                                raise NotFound()
+                        #     if row is None:
+                        #         raise NotFound()
 
-                            try:
-                                attachment = await (
-                                    GetFile(conn)
-                                ).execute(row[0])
+                        #     try:
+                        #         attachment = await (
+                        #             GetFile(conn)
+                        #         ).execute(row[0])
 
-                                output_stream.writestr(
-                                    f'files/{attachment["conf"]["key"]}',
-                                    attachment['file'].getvalue()
-                                )
+                        #         output_stream.writestr(
+                        #             f'files/{attachment["conf"]["key"]}',
+                        #             attachment['file'].getvalue()
+                        #         )
 
-                            except Exception as ce:
-                                if ce.response['Error']['Code'] == '404':
-                                    not_found.append(conf['key'])
-                                    continue
+                        #     except Exception as ce:
+                        #         if ce.response['Error']['Code'] == '404':
+                        #             not_found.append(conf['key'])
+                        #             continue
 
-                                raise Exception(
-                                    "Error while downloading from AWS"
-                                )
+                        #         raise Exception(
+                        #             "Error while downloading from AWS"
+                        #         )
 
```

Comment:
REQ: Löschen nicht nur auskommentieren. Wir haben den Code ja immer noch in Git.

Reply:
Meine Überlegung war hier folgende: Der Export wird ja in naher Zukunft neu erwarbeitet, funktioniert aber rudimentär bisher weiter. Ich habe ihn kurz getestet und konnte keine Funktionsänderung feststellen wenn ich die Getfile Methode lösche. Da es aber keine Teste etc. gibt bin ich mir nicht sicher ob meine Beobachtung zutrifft. Habe es daher auskommentiert drin gelassen damit wir, falls es an der Stelle irgendwie noch ein Problem auftaucht, wissen wo wir ansetzen können. 
Ist aber vermutlich zu weit gedacht. Oder man müsste noch einen Kommentar dazu machen.

Reply:
Ich würde hier einen Kommentar mit Verlinkung auf die PR in diesem kommenden Issue machen. Dann haben wir die Verlinkung gleich dort wo wir sie auch wieder brauchen werden.

Reply:
Gute Idee!